processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/vector/GroupByVectorColumnProcessorFactory.java

```diff
@@ -64,7 +65,16 @@ public GroupByVectorColumnSelector makeMultiValueDimensionProcessor(
         ValueType.STRING == capabilities.getType(),
         "groupBy dimension processors must be STRING typed"
     );
-    throw new UnsupportedOperationException("Multi-value dimensions not yet implemented for vectorized groupBys");
+    throw new UnsupportedOperationException("Multi-value dimensions are not yet implemented for vectorized groupBys");
+  }
+
+  @Override
+  public GroupByVectorColumnSelector makeObjectProcessor(
+      ColumnCapabilities capabilities,
+      VectorObjectSelector selector
+  )
+  {
+    throw new UnsupportedOperationException("Object columns are not yet implemented for vectorized groupBys");
```

Comment:
yes, I have a related branch that I will open as a follow-up to this PR that adds support for group-by queries on vectorized string expressions, using an object selector and a newly added dictionary building `GroupByVectorColumnSelector` similar to the non-vectorized group by code-path. In that branch this part checks that the column capabilities are string typed, and if so then will make the new dictionary building selector, else throw an exception for being an unsupported type.

It seems beneficial to not have to make the string expression vector results have to masquerade as dictionary encoded, to be translated to string values to be re-encoded into a new dictionary, which is part of my reason for pushing for object selectors for the use case

Reply:
Ah, in that case I wouldn't call them "object columns". They're really string columns that are just being accessed through an object selector instead of a dimension selector.